dynatrace-aws-log-forwarder-template.yaml

```diff
@@ -30,6 +30,27 @@ Parameters:
     Default: 'false'
     Description: "If false, expect target ActiveGate to have a self-signed SSL certificate and do not verify its validity"
     Type: String
+  UseExistingActiveGate:
+    Type: String
+    AllowedValues:
+      - 'true'
+      - 'false'
+  DynatracePaasToken:
+    Description: Dynatrace PaaS token
+    Type: String
+    NoEcho: true
+  TenantIdNewAGonly:
+    Type: String
+    Default: "not_provided"
+    Description: "Needed to construct full target-url path in scenario with new AG deployment (with existing AG option, target-url is already provided by DynatraceEnvironmentUrl)"
+  LatestAmazonLinuxAmiId:
+    # this will query for latest AMI ID automatically
+    # https://aws.amazon.com/blogs/compute/query-for-the-latest-amazon-linux-ami-ids-using-aws-systems-manager-parameter-store/
+    Type: 'AWS::SSM::Parameter::Value<AWS::EC2::Image::Id>'
+    Default: '/aws/service/ami-amazon-linux-latest/amzn2-ami-hvm-x86_64-gp2'
+
+Conditions:
+  DeployVPCwithAG: !Equals [ !Ref UseExistingActiveGate, 'false' ]
```

Comment:
imho it's more DeployAGwithVPC, the way it is makes it bit harder to read the conditions later on

Reply:
Maybe DeployAGandVPC? Then in future we can add DeployAGintoExistingVPC?

Reply:
My thinking was more along these lines
DeployAGwithVPC > to me sounds like deploy AG and deploy VPC
DeployAGtoExistingVPC

but DeployAGandVPC is already good

Reply:
Updated to DeployAGwithVPC